### ISSUE TITLE
Can't find variable: screen

### ISSUE THREAD

**Issue by indapublic** [infinitered/reactotron#830]
It works 

```
import Reactotron from 'reactotron-react-native';

Reactotron
  .configure()
  .useReactNative()
  .connect();
```

But if I will add `trackGlobalErrors` or `openInEditor`, it will fail

import Reactotron from 'reactotron-react-native';
import { trackGlobalErrors, openInEditor } from 'reactotron-react-js';

```
Reactotron
  .configure()
  .useReactNative()
  .use(trackGlobalErrors({
  }))
  // .use(openInEditor())
  .connect();
```

<img width="228" alt="screen shot 2018-09-20 at 14 18 04" src="https://user-images.githubusercontent.com/856260/45795635-ffcaa680-bcdf-11e8-992a-029513fbfb79.png">

**Comment by rmevans9** [comment]
Hey there!

When you call .useReqctNative it actually wires those two plugins up for you so there should be no need to import them directly and add them manually. Technically it wouldn’t hurt if you did that so that brings us to the question of why it is blowing up when you do? It’s because you are importing the reactotron-react-js version of them and trying to use them in a react-native environment.

TL;DR just use .useReactNative()
